controlplane/pkg/apis/registry/gen.go

```diff
@@ -1,3 +1,3 @@
 package registry
 
-//go:generate protoc -I . registry.proto --go_out=plugins=grpc:. --proto_path=$GOPATH/src
+//go:generate protoc -I ../../../../vendor -I . registry.proto --go_out=plugins=grpc:. --proto_path=$GOPATH/src
```

Comment:
I don't like this approach but that's something working. Could there be a better way to point protocol to the `vendor` folder?

Reply:
So interestingly, I don't have to do this when I use vendored things:
https://github.com/ligato/networkservicemesh/blob/master/controlplane/pkg/apis/local/connection/connection.proto#L23

https://github.com/ligato/networkservicemesh/blob/master/controlplane/pkg/apis/local/connection/gen.go#L3

What problem did you hit that led to this?